.mina/telegram.toml

```
@@ -0,0 +1,21 @@
+includes = ["avilla/telegram"]
+raw-dependencies = [
+  "avilla-core >= 1.0.0a8",
+  "avilla-standard-telegram >= 0.1.0"
+]
+
+[project]
+name = "avilla-telegram"
+version = "0.1.0" # WIP for release.
+authors = [
+  {name = "nullqwertyuiop", email = "null@member.fsf.org"},
+]
+dependencies = []
```

依赖是不是忘记了？

---

```suggestion
dependencies = [
  "aiohttp"
]
```

应该就没了？原先的 `python-telegram-bot` 不用了